aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
```

Comment:
If you want to add a link to your name, I'll suggest doing it in the [`authors.txt` file](https://github.com/aspnet/Docs/blob/master/common/authors.txt) instead